py/trtorch/csrc/utils.h

```diff
@@ -0,0 +1,138 @@
+#pragma once
+#include <pybind11/numpy.h>
+#include <pybind11/pybind11.h>
+
+#include <functional>
+#include <iostream>
+#include <string>
+#include "NvInfer.h"
+
+namespace tensorrt {
+namespace utils {
+
+namespace py = pybind11;
+
+// Returns the size in bytes of the specified data type.
+inline size_t size(nvinfer1::DataType type) {
+  switch (type) {
+    case nvinfer1::DataType::kFLOAT:
+      return 4;
+    case nvinfer1::DataType::kHALF:
+      return 2;
+    case nvinfer1::DataType::kINT8:
+      return 1;
+    case nvinfer1::DataType::kINT32:
+      return 4;
+    case nvinfer1::DataType::kBOOL:
+      return 1;
+  }
+  return -1;

```

Comment:
We should just throw an error here and return 0 to suppress any compiler warnings. Returning -1 for an unsigned type is probably dangerous

Reply:
Fixed it. This was inherited from TRT source code. I cleaned it up  to be as minimal and relevant as possible.